Sources/TensorFlow/Operators/Math.swift

```diff
@@ -2686,6 +2686,33 @@ internal extension Tensor where Scalar: TensorFlowFloatingPoint {
     }
 }
 
+/// Returns the Cholesky decomposition of one or more square matrices.
+///
+/// The input is a tensor of shape `[..., M, M]` whose inner-most 2 dimensions
+/// form square matrices.
+///
+/// The input has to be symmetric and positive definite. Only the lower-triangular
+/// part of the input will be used for this operation. The upper-triangular part
+/// will not be read.
+///
+/// The output is a tensor of the same shape as the input
+/// containing the Cholesky decompositions for all input submatrices `[..., :, :]`.
+///
+/// - Parameter input: A tensor of shape `[..., M, M]`.
+@inlinable
+@differentiable(vjp: _vjpCholesky)
+public func cholesky<T: TensorFlowFloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
```

Comment:
Does it work with batching?

Reply:
Yup, it sure does! Check out some of the shapes used in testing.
https://github.com/tensorflow/swift-apis/blob/c09de96f739aa8bfce0c41e19213bd2d7fa9a4ba/Tests/TensorFlowTests/OperatorTests/MathTests.swift#L488